### ISSUE TITLE
bullmq@1.59.0: ScriptLoaderError: No .lua files found!

### ISSUE THREAD

**Issue by Niek** [taskforcesh/bullmq#949]
I just bumped my dependencies to BullMQ 1.59.0, and get this error:
```js
node:internal/process/promises:246
          triggerUncaughtException(err, true /* fromPromise */);
          ^

ScriptLoaderError: No .lua files found!
    at ScriptLoader.loadScripts (node_modules/bullmq/dist/cjs/commands/script-loader.js:283:19)
    at async ScriptLoader.load (node_modules/bullmq/dist/cjs/commands/script-loader.js:308:29)
    at async RedisConnection.init (node_modules/bullmq/dist/cjs/classes/redis-connection.js:100:9)
Emitted 'error' event on Queue instance at:
    at Queue.emit (node_modules/bullmq/dist/cjs/classes/queue-base.js:38:30)
    at RedisConnection.emit (node:events:390:28)
    at node_modules/bullmq/dist/cjs/classes/redis-connection.js:45:45 {
  includes: [],
  line: 0,
  position: 0
}
```

I think this might be a side-effect of https://github.com/taskforcesh/bullmq/pull/947 that was merged.

**Comment by manast** [comment]
Interesting, ok, fix coming soon.

**Comment by Niek** [comment]
@manast see also my comment here: https://github.com/taskforcesh/bullmq/issues/901#issuecomment-986693771 regarding using `import` rather than `readFile`.

**Comment by Niek** [comment]
I can confirm it's fixed in 1.59.1, thanks!

**Comment by jiangdi0924** [comment]
1.59.2 ; throw error:  No .lua files found!
